test/e2e/horizontal_pod_autoscaling.go

```diff
@@ -44,6 +42,7 @@ var _ = Describe("Horizontal pod autoscaling (scale resource: CPU)", func() {
 	// TODO(madhusudancs): Fix this when Scale group issues are resolved (see issue #18528).
 	// These tests take ~20 minutes each.
 	// Describe("[Serial] [Slow] Deployment", func() {
+	// KubeDescribe("Deployment", func() {
```

Comment:
yes ... we can remove the `Describe` here.  I wasn't sure what to do since the comment obviously meant to refer to an artifact from the past.  but I guess removing would be the best.